src/lib.rs

```diff
@@ -702,6 +711,24 @@ macro_rules! create_app {
                             .long("private-key")
                             .help("Local path to developer's Eliptic Curve private key.")
                             .takes_value(true),
+                    )
+                    .arg(
+                        Arg::with_name("image_name")
```

Comment:
Add a `default` value here as well.

Reply:
Do you mean add a description of what will be taken as a default?
If you mean the value of the argument itself, then it's `Optional<String>` and will be propagated this way up to `Docker2Eif::new` where in case of `None` will be replaced with docker tag (or hash as discussed somewhere here)